h5p-bildetema-words-topic-image/language/.en.json

```
@@ -3,6 +3,9 @@
     {
       "label": "Image"
     },
+    {
+      "label": "Backend Url"
```

This can also be added, similar as `.en.json` for h5p-bildetema

```suggestion
      "label": "Backend Url",
      "description": "The Url to the json database",
      "default": "https://cdn-devbildetema.azureedge.net/data/database.json"
```